packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -885,6 +886,20 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
     const newValue = newProps.value;
     workInProgress.memoizedProps = newProps;
 
+    if (__DEV__) {
+      const providerPropTypes = workInProgress.type.propTypes;
+      const {getCurrentFiberStackAddendum} = ReactDebugCurrentFiber;
```

Comment:
Please move this right after imports so we only do this once. You can check how we do this in other files.